src/test/java/pl/project13/maven/git/GitCommitIdMojoTest.java

```diff
@@ -27,6 +27,7 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Paths;
 import java.util.Map;
```

Comment:
Yeah... I'm thinking that maybe it's time to drop JDK6... I mean, JDK7 is OOL in a second.

Let's drop 6 I think.